src/sentry/api/endpoints/organization_events.py

```diff
@@ -188,3 +189,32 @@ def get(self, request, organization):
             # in snuba
             'count': data['count'] * 10,
         })
+
+
+class OrganizationEventJsonView(OrganizationEventsEndpointBase):
+    # TODO(lb): should permissions be any different here?
+    # required_scope = 'event:read'
+
+    def get(self, request, organization, event_id):
+        # TODO(lb): This endpoint used to handle both snuba and without snuba.
+        # Couldnt' think of a reason why we wanted without snuba
+
+        filter_params = self.get_filter_params(request, organization)
+        project_id = filter_params.get('project_id')
+        if not project_id:
+            return Response({'data': []})
+
+        # TODO(lb): this checks both hex event_id or postgres primary key
```

Comment:
@markstory do you know what is being sent here? Are there any postgres `event.id` stuff going on or can you just send the snuba event hex?

Reply:
I would like to send the snuba hex id.